### PR TITLE
users: Print details about user teams

### PR DESCRIPTION
**Pull request by mike-sul**
Print scopes and device groups of each user team.

### CODE REVIEW

**Comment by mike-sul** [comment]
This can be merged only after this https://git.foundries.io/development/cloudplatforms/infrastructure/ota-lite/-/merge_requests/403 is deployed.

**Review by doanac — APPROVED** [review]

**Comment by mike-sul** [comment]
@doanac It could be something more vertical and a factory name is removed from each scope.

```
fioctl users  5d933baefae079001d6ecf5e
ID                        NAME           ROLE
--                        ----           ----
5d933baefae079001d6ecf5e  Mike Sul User  member

TEAMS
-----
test-team-01
  Scopes
  ------
    ci:read
    devices:read-update

  Groups
  ------
    user-dev-grp
    new-dev-group-stag

test-team-03
  Scopes
  ------
    devices:create
    devices:delete


EFFECTIVE SCOPES
----------------
msul-dev01:ci:read
msul-dev01:devices:create
msul-dev01:devices:delete
msul-dev01:devices:read-update

```

**Comment by doanac** [comment]
I think the compact is fine. It also makes it a little easier for someone to do something like `.. | grep Scopes`

File: subcommands/users/cmd.go [hunk 1]
```diff
@@ -2,12 +2,12 @@ package users
 
 import (
 	"github.com/cheynewallace/tabby"
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
```
Comment (doanac) [inline subcommands/users/cmd.go:6]: Normally golang IDEs put your own imports at the bottom like they were before.

Comment (mike-sul) [reply subcommands/users/cmd.go:6]: `make format` does it, not IDE.

Comment (doanac) [reply subcommands/users/cmd.go:6]: OMG - now i know why your golang files differ from how @vkhoroz and I do them. 

Volodya - what do you think?

**Review by vkhoroz — APPROVED** [review]
LGTM

**Comment by mike-sul** [comment]
```
```
fioctl users  5d933baefae079001d6ecf5e
ID                        NAME           ROLE
--                        ----           ----
5d933baefae079001d6ecf5e  Mike Sul User  member

TEAMS
-----
test-team-01
  Scopes: ci:read, devices:read-update
  Groups: user-dev-grp, new-dev-group-stag

test-team-03
  Scopes: devices:create, devices:delete


EFFECTIVE SCOPES
----------------
ci:read
devices:create
devices:delete
devices:read-update

```
```

File: subcommands/users/cmd.go (hunk 1 above)
Comment (vkhoroz) [reply subcommands/users/cmd.go:6]: I simply use `make format` before every commit. That's easy.

BTW. the go-linter should fail if there are format errors, isn't it?

Comment (vkhoroz) [reply subcommands/users/cmd.go:6]: I mean, IDEs differ (e.g. with Vim the only IDE is my brain); make format is uniform.

Comment (doanac) [reply subcommands/users/cmd.go:6]: i have a feeling i've set our formatter up wrong. I don't think the linter cares about the import order. anyway - i guess  `make format` rules for now.

**Review by doanac — APPROVED** [review]

**Pull request merged**